### ISSUE TITLE
Heavy CPU usage

### ISSUE THREAD

**Issue by Tangerino** [sourcery-ai/sourcery#100]
Machine start to get slow, looking into the task manager I found this

```
ps ax | grep sourcery
 3671   ??  R     11:32.29 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3682   ??  R     10:19.65 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3693   ??  R      9:15.61 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3712   ??  R      8:35.48 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3742   ??  R      7:52.11 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3756   ??  R      7:07.67 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3767   ??  R      5:04.08 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3808   ??  R      4:19.72 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3818   ??  R      3:39.04 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3827   ??  R      2:59.18 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3839   ??  R      2:31.96 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3846   ??  R      2:04.30 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3853   ??  R      1:40.85 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
 3861   ??  R      1:18.64 /Users/carlos/Library/Application Support/JetBrains/PyCharm2020.2/plugins/sourcery/lib/mac/sourcery legacy-server
```

Still running after shutdown PyCharm

**Comment by Tangerino** [comment]
I think this is related to #92

**Comment by Hellebore** [comment]
Looks like it is related - thanks for the extra information! This should definitely help me to track it down.

**Comment by Hellebore** [comment]
Have identified some possible improvements - we're currently reviewing them.

**Comment by Hellebore** [comment]
Potential fix is now live with version 0.8.4 - please re-open if it doesn't address the issue.
